components/LabeledLinkButton.jsx

```diff
@@ -0,0 +1,22 @@
+import StyledShowMore from '../components-styled/StyledShowMore';
+import StyledSvgWrapper from '../components-styled/StyledSvgWrapper';
+import Link from 'next/link';
+
+export default function LabeledLinkButton({
+  label,
+  target,
+  children,
+  onClick,
+  className,
+}) {
+  return (
+    <Link passHref href={`/${target}`}>
+      <StyledShowMore onClick={onClick} className={`${className}`}>
```

Comment:
I guess you can hand over the className directly instead of wrapping it into the template literal. (in fact you could use the rest and spread operator instead of handing over onClick and className separately, but thats up to you:
```js
label,
target,
children,
...remainingProps
})
.......
<StyledShowMore {...remainingProps}>
```

Reply:
done